### PR TITLE
Updated scripts and k8s config files for base ibm fabric 1.1 images

### PR DESCRIPTION
**Pull request by anillewis**
Updated the scripts and k8s config files for enabling base ibm fabric 1.1 images to work with couchdb (free plan). Had to change the example for chaincode install and instantiate as chaincode_example02 does not work. Note that composer still has issues but the base fabric network can be started (you need to comment create/create_composer-playground.sh $@ in create_all.sh for this to work)

### CODE REVIEW

File: cs-offerings/kube-configs/chaincode_install.yaml.base [hunk 1]
```diff
@@ -13,7 +13,7 @@ spec:
   - name: chaincodeinstall
     image: ibmblockchain/fabric-tools:1.1.0
     imagePullPolicy: Always
-    command: ["sh", "-c", "git clone -b v1.1.0 https://github.com/hyperledger/fabric $GOPATH/src/github.com/hyperledger/fabric/ && peer chaincode install -n ${CHAINCODE_NAME} -v ${CHAINCODE_VERSION} -p github.com/hyperledger/fabric/examples/chaincode/go/chaincode_example02/"]
+    command: ["sh", "-c", "git clone -b release-1.1 https://github.com/hyperledger/fabric-samples $GOPATH/src/github.com/hyperledger/fabric-samples/ && peer chaincode install -n ${CHAINCODE_NAME} -v ${CHAINCODE_VERSION} -p github.com/hyperledger/fabric-samples/chaincode/sacc"]
```
Comment (mrshah-at-ibm) [inline cs-offerings/kube-configs/chaincode_install.yaml.base]: Can you clone `v1.1.1` and not `release-1.1` so that we point to releases only

Comment (mrshah-at-ibm) [reply cs-offerings/kube-configs/chaincode_install.yaml.base]: Also I am curios why you moved to `sacc` and not change the example02 location to `https://github.com/hyperledger/fabric/tree/v1.1.1/examples/chaincode/chaintool/example02/src/chaincode`

File: cs-offerings/scripts/delete/delete_channel-pods.sh [hunk 2]
```diff
@@ -18,12 +18,12 @@ echo "Preparing yaml for createchannel pod for deletion"
 sed -e "s/%PEER_ADDRESS%/${PEER_ADDRESS}/g" -e "s/%CHANNEL_NAME%/${CHANNEL_NAME}/g" -e "s/%PEER_MSPID%/${PEER_MSPID}/g" ${KUBECONFIG_FOLDER}/create_channel.yaml.base > ${KUBECONFIG_FOLDER}/create_channel.yaml
 
 echo "Deleting Existing Create Channel Pod"
-if [ "$(kubectl get pods | grep createchannel | wc -l | awk '{print $1}')" != "0" ]; then
+if [ "$(kubectl get pods createchannel | grep createchannel | wc -l | awk '{print $1}')" != "0" ]; then
```
Comment (mrshah-at-ibm) [inline cs-offerings/scripts/delete/delete_channel-pods.sh]: I am curios on why the changes in this file, like the one here, are required?

File: cs-offerings/scripts/delete/delete_chaincode-instantiate.sh [hunk 3]
```diff
@@ -21,12 +21,12 @@ echo "Preparing yaml for chaincode instantiate delete"
 sed -e "s/%CHANNEL_NAME%/${CHANNEL_NAME}/g" -e "s/%PEER_ADDRESS%/${PEER_ADDRESS}/g" -e "s/%PEER_MSPID%/${PEER_MSPID}/g" -e "s|%MSP_CONFIGPATH%|${MSP_CONFIGPATH}|g"  -e "s/%CHAINCODE_NAME%/${CHAINCODE_NAME}/g" -e "s/%CHAINCODE_VERSION%/${CHAINCODE_VERSION}/g" ${KUBECONFIG_FOLDER}/chaincode_instantiate.yaml.base > ${KUBECONFIG_FOLDER}/chaincode_instantiate.yaml
 
 echo "Deleting Existing Instantiate Chaincode Pod"
-if [ "$(kubectl get pods | grep chaincodeinstantiate | wc -l | awk '{print $1}')" != "0" ]; then
+if [ "$(kubectl get pods chaincodeinstantiate | grep chaincodeinstantiate | wc -l | awk '{print $1}')" != "0" ]; then
```
Comment (mrshah-at-ibm) [inline cs-offerings/scripts/delete/delete_chaincode-instantiate.sh]: Why is this change required?

File: cs-offerings/scripts/delete/delete_chaincode-install.sh [hunk 4]
```diff
@@ -21,12 +21,12 @@ echo "Preparing yaml for chaincodeinstall pod"
 sed -e "s/%PEER_ADDRESS%/${PEER_ADDRESS}/g" -e "s/%PEER_MSPID%/${PEER_MSPID}/g" -e "s|%MSP_CONFIGPATH%|${MSP_CONFIGPATH}|g"  -e "s/%CHAINCODE_NAME%/${CHAINCODE_NAME}/g" -e "s/%CHAINCODE_VERSION%/${CHAINCODE_VERSION}/g" ${KUBECONFIG_FOLDER}/chaincode_install.yaml.base > ${KUBECONFIG_FOLDER}/chaincode_install.yaml
 
 echo "Deleting Existing Install Chaincode Pod"
-if [ "$(kubectl get pods | grep chaincodeinstall | wc -l | awk '{print $1}')" != "0" ]; then
+if [ "$(kubectl get pods chaincodeinstall | grep chaincodeinstall | wc -l | awk '{print $1}')" != "0" ]; then
```
Comment (mrshah-at-ibm) [inline cs-offerings/scripts/delete/delete_chaincode-install.sh]: Why is this change required?

File: cs-offerings/scripts/delete/delete_chaincode-instantiate.sh (hunk 3 above)
Comment (anillewis) [reply cs-offerings/scripts/delete/delete_chaincode-instantiate.sh]: I kept it consistent with the version that was committed in the master branch by rddill. I also observed that $(kubectl get pods | grep chaincodeinstantiate wasn't working. I had to use $(kubectl get pods -a | grep chaincodeinstantiate or $(kubectl get pods chaincodeinstantiate for this to work

File: cs-offerings/scripts/delete/delete_chaincode-install.sh (hunk 4 above)
Comment (anillewis) [reply cs-offerings/scripts/delete/delete_chaincode-install.sh]: I kept it consistent with the version that was committed in the master branch by rddill. I also observed that $(kubectl get pods | grep chaincodeinstall  wasn't working. I had to use $(kubectl get pods -a | grep chaincodeinstall or $(kubectl get pods chaincodeinstall for this to work

File: cs-offerings/scripts/delete/delete_channel-pods.sh (hunk 2 above)
Comment (anillewis) [reply cs-offerings/scripts/delete/delete_channel-pods.sh]: I kept it consistent with the version that was committed in the master branch by rddill. I also observed that $(kubectl get pods | grep createchannel wasn't working. I had to use $(kubectl get pods -a | grep createchannel or $(kubectl get pods createchannel for this to work

File: cs-offerings/scripts/delete/delete_chaincode-instantiate.sh (hunk 3 above)
Comment (mrshah-at-ibm) [reply cs-offerings/scripts/delete/delete_chaincode-instantiate.sh]: thanks

File: cs-offerings/scripts/delete/delete_chaincode-install.sh (hunk 4 above)
Comment (mrshah-at-ibm) [reply cs-offerings/scripts/delete/delete_chaincode-install.sh]: thanks

File: cs-offerings/scripts/delete/delete_channel-pods.sh (hunk 2 above)
Comment (mrshah-at-ibm) [reply cs-offerings/scripts/delete/delete_channel-pods.sh]: thanks

File: cs-offerings/kube-configs/chaincode_install.yaml.base (hunk 1 above)
Comment (anillewis) [reply cs-offerings/kube-configs/chaincode_install.yaml.base]: I have updated the config and scripts based on the review comments but instead of https://github.com/hyperledger/fabric/tree/v1.1.1/examples/chaincode/chaintool/example02/src/chaincode I have used github.com/hyperledger/fabric/examples/chaincode/go/chaincode_example02. Earlier I wasn't able to identify why chaincode_example02 was failing to instantiate (the install would always work) but figured out now that the chaincode name had to be updated from example02 to chaincode_example02 to get this working. Please pull my latest updates for review

File: cs-offerings/kube-configs/blockchain-couchdb.yaml [hunk 5]
```diff
@@ -350,7 +354,7 @@ spec:
     spec:
       containers:
       - name: couchdb1
-        image: ibmblockchain/fabric-couchdb:0.4.6
+        image: ibmblockchain/fabric-couchdb:0.4.10
```
Comment (davidkel) [inline cs-offerings/kube-configs/blockchain-couchdb.yaml]: I believe this is the wrong version of fabric-couch for fabric 1.1. 0.4.6 is for fabric 1.1, 0.4.10 is for fabric 1.2

Comment (anillewis) [reply cs-offerings/kube-configs/blockchain-couchdb.yaml]: As per https://raw.githubusercontent.com/hyperledger/fabric/master/scripts/bootstrap.sh which is referenced in https://hyperledger-fabric.readthedocs.io/en/release-1.1/samples.html, third party images are tagged at 0.4.10

Comment (davidkel) [reply cs-offerings/kube-configs/blockchain-couchdb.yaml]: The bootstrap script is in master and is for all versions of fabric. The default for that script is for the current release of 1.2 which pre-reqs 0.4.10. For 1.1 the pre-req version is 0.4.6.
see https://github.com/hyperledger/fabric/blob/095f8815a8ea1e3450595a76e0a464416a145692/Makefile#L47
which is the fabric 1.1 makefile for example.

Comment (anillewis) [reply cs-offerings/kube-configs/blockchain-couchdb.yaml]: Updated the version based on the above reviewer comment. Please pull the latest version for review
